scripts/genvp.py

```diff
@@ -2281,9 +2281,18 @@ def gen_structCompare(self, fmt, structDef, var, values, parentLimittype = None)
                 elif limittype == 'max':
                     # Compare max limit by checking if device value is greater than or equal to profile value
                     comparePredFmt = '{0} >= {1}'
+                elif limittype == 'bits':
+                    # Behaves like max, but smaller values are allowed, so no check needed
+                    comparePredFmt = None
```

Comment:
I am not sure this is quite right. `comparePredFmt = '{0} >= {1}'` was probably right here.

Reply:
But aren't smaller values allowed?